data/virt_autotest/host_unattended_installation_files/agama/sle16.jsonnet

```diff
@@ -0,0 +1,110 @@
+{
+  product: {
+    id: 'SLES'
+  },
+  user: {
+    userName: 'bernhard',
+    fullName: "Bernhard M. Wiedemann",
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true
+  },
+  root: {
+    password: '$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/',
+    hashedPassword: true
+  },
+  legacyAutoyastStorage: [
+      {
+         "device": "/dev/sda",
+         "disklabel": "gpt",
+         "enable_snapshots": false,
+         "initialize": true,
+         "partitions": [
+            {
+               "create": true,
+               "filesystem": "vfat",
+               "format": true,
+               "mount": "/boot/efi",
+               "mountby": "uuid",
+               "partition_id": 259,
+               "size": "512M"
+            },
+            {
+               "create": true,
+               "create_subvolumes": true,
+               "filesystem": "btrfs",
+               "format": true,
+               "mount": "/",
+               "mountby": "uuid",
+               "partition_id": 131,
+               "size": "120G"
+            },
+            {
+               "create": true,
+               "filesystem": "xfs",
+               "format": true,
+               "mount": "/var/lib/libvirt/images/",
+               "mountby": "uuid",
+               "partition_id": 131,
+               "resize": false
+            },
+            {
+               "create": true,
+               "filesystem": "swap",
+               "format": true,
+               "mountby": "uuid",
+               "size": "2G"
```

Comment:
Is it fast enough for large amount size data IO with 2G swap, eg vm disk copying? Maybe we can consider to bump it to 4G for those bare-metalx machines, if it is not fast enough.

Reply:
'2G' is the default size for swap partition on sle15. I converted the storage configuration to sle16 directly. I am not familiar with this size.  Hi team, anyone has idea if 2G or 4G is better for virt tests?

Reply:
Let me run a job to check the default size on SLE16. a moment.

Reply:
Here is the default partitions with kvm_role on sle16:

```
qa2-129-227:~ # lsblk
NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINTS
sda      8:0    0 838.4G  0 disk 
├─sda1   8:1    0   512M  0 part /boot/efi
├─sda2   8:2    0 835.9G  0 part /boot/grub2/x86_64-efi
│                                /home
│                                /root
│                                /srv
│                                /opt
│                                /usr/local
│                                /boot/grub2/i386-pc
│                                /.snapshots
│                                /var
│                                /
└─sda3   8:3    0     2G  0 part [SWAP]
```

Reply:
Personally, I usually set 4G swap on my personal computer :) . Will let you decide. There is always chance to update once needed in future PRs, so no big deal.

Reply:
> Will let you decide. 

I won't decide as I know little about swap. You guys decide, I just make the change based on your decision.

Reply:
Haha, okay, I recommend 4G swap :)

Reply:
No problem.